templates/config.py.j2

```diff
@@ -0,0 +1,22 @@
+USERNAME = "{{ mattermost_github_integration_username }}"
+ICON_URL = "{{ mattermost_github_integration_icon_url }}"
+MATTERMOST_WEBHOOK_URLS = {
+    {% for item in mattermost_github_integration_webhook_urls %}
+        '{{ item.name }}' : ("{{ item.url }}", "{{ item.channel }}"),
```

Comment:
Is this format valid with the trailing comma?

Reply:
yes